### PR TITLE
added run_git to config

### PR DESCRIPTION
**Pull request by SeanTConrad**
And cleaned up sample YAML

### CODE REVIEW

File: src/verinfast/config.py [hunk 1]
```diff
@@ -423,6 +423,9 @@ def handle_config_file(self):
             if "should_upload" in self.config:
                 self.shouldUpload = self.config["should_upload"]
 
+            if "run_git" in self.config:
+                self.runGit = self.config["run_git"]
+
```
Comment (SeanTConrad) [inline src/verinfast/config.py:428]: I'm not 100% certain how this plays with the ARGS and other places runGit gets set.

File: src/verinfast/config.yaml [hunk 2]
```diff
@@ -2,38 +2,32 @@
 
 baseurl: https://beta.startupos.dev/api/v2
 # baseurl: http://localhost:4000
-should_upload: true
+should_upload: true # Optional parameter to turn upload off or on. Default is true
+run_git: false # Optional parameter to turn git scan off or on. Default is true
```
Comment (aylusltd) [inline src/verinfast/config.yaml:6]: I think default state is false to prevent breaking local repos.

File: src/verinfast/config.yaml [hunk 3]
```diff
@@ -2,38 +2,32 @@
...
     dry: true # Flag to just reupload files
     dependencies: true
-#   cloud:
-#   - provider: azure
-# #     # No idea wtf this is
-# #     # account: d6f7992c-2a21-4a9c-b46b-e70d2c90070d
-# #     # SOS account
-#     account: 80dc7a6b-df94-44be-a235-7e7ade335a3c
-#     start: "2023-05-01"
-#     end: "2023-09-01"
```
Comment (aylusltd) [inline src/verinfast/config.yaml:22]: At least move this to its own conf so we can add the unit tests we're missing.

File: src/verinfast/config.py (hunk 1 above)
Comment (aylusltd) [reply src/verinfast/config.py:428]: config is read first and then overwritten by args. runGit is also set true if there's a repo in the conf that isn't local.

**Review by aylusltd — CHANGES_REQUESTED** [review]

**Review by aylusltd — APPROVED** [review]

**Pull request merged**